design/www/app/login/login.controller.js

```diff
@@ -0,0 +1,100 @@
+(function () {
+    'use strict';
+
+    angular.module('starter')
+            .controller('loginController', loginController);
+
+    function loginController($state, loginFactory, localStorageService, $localStorage, tostService, deviceService, googleLogin, facebookLogin) {
+        console.log('login');
+        this.data = {
+            email: '',
+            password: ''
+        }
+        // var deviceUUID = deviceService.getuuid();
+        var deviceUUID = 'device_id';
```

Comment:
this is only for the test in web browser. i created a service for get deviceUUID.